### PR TITLE
Parser refactoring and interfaceId introduction

### PR DESCRIPTION
**Pull request by radoslavvlaskovski**
Unused parameters where deleted and the interfaceId was introduced

### CODE REVIEW

File: tosca-parser/src/main/java/org/openbaton/tosca/parser/CSARParser.java [hunk 1]
```diff
@@ -17,19 +17,8 @@
...
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.io.*;
```
Comment (lorenzotomasini) [inline tosca-parser/src/main/java/org/openbaton/tosca/parser/CSARParser.java]: Let's keep all static imports, avoiding using *

File: tosca-parser/src/main/java/org/openbaton/tosca/parser/CSARParser.java [hunk 2]
```diff
@@ -290,7 +261,7 @@ public NetworkServiceDescriptor onboardNSD(byte[] bytes, String projectId)
 
     for (VirtualNetworkFunctionDescriptor vnfd : nsd.getVnfd()) {
       if (!folderNames.contains(vnfd.getType())) {
-        throw new NotFoundException("No Scripts specified for the VNFD of type: " + vnfd.getType());
+        log.info("No Scripts specified for the VNFD of type: " + vnfd.getType());
```
Comment (lorenzotomasini) [inline tosca-parser/src/main/java/org/openbaton/tosca/parser/CSARParser.java]: warning would be better

File: tosca-parser/src/main/java/org/openbaton/tosca/templates/TopologyTemplate/TopologyTemplate.java [hunk 3]
```diff
@@ -127,14 +118,6 @@ public void setNode_templates(Map<String, NodeTemplate> node_templates) {
...
-    return "Topology: \n"
-        + "inuts: "
-        + inputs
-        + "\n"
-        + "substitution_mappings: "
-        + substitution_mappings
-        + "\n"
-        + "Nodes: \n"
-        + node_templates;
+    return "Topology: \n" + "inuts: " + inputs + "\n" + "Nodes: \n" + node_templates;
```
Comment (lorenzotomasini) [inline tosca-parser/src/main/java/org/openbaton/tosca/templates/TopologyTemplate/TopologyTemplate.java]: please use intellij idea template for toStrings

**Comment by radoslavvlaskovski** [comment]
@lorenzotomasini should be ok now

**Pull request merged**